stat/roc.go

```diff
@@ -141,7 +139,7 @@ func ROC(cutoffs, y []float64, classes []bool, weights []float64) (tpr, fpr, thr
 // The returned ntp values can be interpreted as the number of true positives
 // where values above the given rank are assigned class true for each given
 // rank from 1 to len(classes).
```

Comment:
Is it true that `ntp[i]` corresponds to rank `len(classes) - i`? If yes, then maybe we should mention that for clarity.

Reply:
No, it corresponds to the sum of positive weights.

Reply:
This is addressed in the paragraph after the formula though (which had a residual incorrectness — now fixed).